user-guide/modules/ROOT/pages/getting-started.adoc

```diff
@@ -0,0 +1,886 @@
+
+This is the usual procedure to configure, build and install Boost with boost:build[]:
+
+Decide where you are going to install the libraries to, most locations are OK, something like *Documents/Boost-libraries* is fine.
+If your location has a space in the name, you will need to surround the path in quotes in the commands that follow.
+
+Windows:
+
+[source]
+----
+bootstrap.bat
+b2
+b2 install --prefix=C:\Boost
+----
+
+If you get an error during the build process such as the following, you probably do not have an up to date and full installation of pass:[C++] on your machine.
+Go back to <<Prerequisites>> and install or update your version of Visual Studio.
+
+image::cpp-runtime-error.png[]
+
+Unix variants:
+
+[source]
+----
+./bootstrap.sh
+./b2
+./b2 install --prefix=/usr/local
+----
+
+You can adjust the installation `prefix` to your preference in this command.
+
+NOTE:: We break the build and install into separate steps, in order to get better diagnostics when building the libraries, and to suppress too many messages when installing the libraries.
+The single command `.\b2.exe --prefix=<YOUR-INSTALL-FOLDER> -d0 install` will both build and install, but suppresses too many messages during the build process.
+If something goes wrong, it will probably be during the build process, and we want to know about it.
+
+The build process creates all variants of each library.
+For example, there may be _Release_ and _Debug_ variants, or possibly _Multithreaded_ and _Static_ runtime variants.
+Also, libraries that are _header-only_ (in other words, do not require being built) are copied to your installation folder.
+
+TIP: Consider using the `--show-libraries` and `--with-libraries=library-name-list` options if you want to limit the wait instead of building everything.
+--
+
+CMake::
++
+--
+You can also build the Boost libraries with CMake.
+This is the usual procedure to build and install Boost with CMake:
+
+[source]
+----
+mkdir __build
+cd __build
+cmake ..
+cmake --build .
+cmake --build . --target install
+----
+
+If you are using a multi-config generator, such as Visual Studio, you can install the Debug and Release variants of the library separately with:
+
+[source]
+----
+cmake --build . --target install --config Debug
+cmake --build . --target install --config Release
+----
+
+Although both the boost:build[] and the CMake scripts work fine, the final installed libraries will have different layouts for historical reasons.
+Both the CMake integration scripts and the libraries will be installed under different paths and this can create incompatibilities between Boost installations.
+
+TIP: You don't need to *build* Boost with CMake to be able to *use* Boost with CMake.
+CMake integration will work fine even if you install Boost with boost:build[].
+
+The supported and tested way to build Boost remains with boost:build[] and you should refrain from using CMake at this step unless you need this functionality.
```

Comment:
What does "this functionality" mean in this context? If we're presenting this as something advanced, it shouldn't be in the getting started, in my opinion.